src/NewsService.Models.Dto/Requests/Filters/FindNewsFilter.cs

```diff
@@ -1,20 +1,23 @@
-﻿using Microsoft.AspNetCore.Mvc;
-using System;
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
 
 namespace LT.DigitalOffice.NewsService.Models.Dto.Requests.Filters
 {
-    public class FindNewsFilter
-    {
-        [FromQuery(Name = "authorid")]
-        public Guid? AuthorId { get; set; }
+  public class FindNewsFilter
+  {
+    [FromQuery(Name = "authorid")]
+    public Guid? AuthorId { get; set; }
 
-        [FromQuery(Name = "departmentid")]
-        public Guid? DepartmentId { get; set; }
+    [FromQuery(Name = "departmentid")]
+    public Guid? DepartmentId { get; set; }
 
-        [FromQuery(Name = "pseudonym")]
-        public string Pseudonym { get; set; }
+    [FromQuery(Name = "pseudonym")]
+    public string Pseudonym { get; set; }
 
-        [FromQuery(Name = "subject")]
-        public string Subject { get; set; }
-    }
+    [FromQuery(Name = "subject")]
```

Comment:
serch by subject should be in serch service

Reply:
done